From: Andreas Gruenbacher <agruen@suse.de>
Subject: Fix "make kernelrelease" to work without "make prepare"

Without this change, "make kernelrelease" reports an empty string
until "make prepare" has creates .kernelrelease. There is no real
reason why "make kernelrelease" should not work for an unconfigured
kernel; this change fixes it.

Signed-off-by: Andreas Gruenbacher <agruen@suse.de>

Index: linux-2.6.15/Makefile
===================================================================
--- linux-2.6.15.orig/Makefile
+++ linux-2.6.15/Makefile
@@ -1314,7 +1314,7 @@ checkstack:
 	$(PERL) $(src)/scripts/checkstack.pl $(ARCH)
 
 kernelrelease:
-	@echo $(KERNELRELEASE)
+	@echo $(kernelrelease)
 kernelversion:
 	@echo $(VERSION).$(PATCHLEVEL).$(SUBLEVEL)$(EXTRAVERSION)
 
